src/crawlee/crawlers/_playwright/_playwright_crawler.py

```diff
@@ -114,7 +114,8 @@ def __init__(
             browser_pool: A `BrowserPool` instance to be used for launching the browsers and getting pages.
             user_data_dir: Path to a user data directory, which stores browser session data like cookies
                 and local storage.
-            browser_type: The type of browser to launch ('chromium', 'firefox', or 'webkit').
+            browser_type: The type of browser to launch ('chromium', 'firefox', 'webkit' or 'chrome'). Use `chrome` to
+                use the installed Chrome browser instead of Chromium.
```

Comment:
If I understand correctly, `chromium`, `firefox`, and `webkit` are Playwright's bundled browsers, but when the `chrome` option is used, it relies on the user's installed Chrome instance. If that's correct, could you please update the docstring to clarify this?